docs/nx-cloud/reference/launch-templates.md

```diff
@@ -55,24 +59,262 @@ The following resource classes are available:
 - `docker_linux_arm64/extra_large`
 - `windows/medium`
 
-See their detailed description and pricing at [nx.app/pricing](https://nx.app/pricing).
+See their detailed description and pricing at [nx.app/pricing](https://nx.app/pricing#plan-detail?sutm_source=nx.dev&utm_medium=launch-templates).
 
-## Image
+### `launch-templates.<template-name>.image`
 
-The following images are available:
+The `launch-templates.<template-name>.image` defines the available base software for the agent machine.
+
+```yaml {% fileName=".nx/workflows/agents.yaml" %}
+launch-templates:
+  template-one:
+    image: 'ubuntu22.04-node20.11-v7'
+```
+
+Nx Cloud provides the following images:
```

Comment:
I don't know that we want to list every image here. Most of the updates we have published are out of necessity, so we probably don't want someone to pick an arbitrary older version for whatever reason.

We might just want to list the most up-to-date version and revisit the changelog/feature lists as a separate item.

Reply:
I'll just list from v5 onwards since that's the ones we actually have "notes" for. a more verbose changelog/release notes will be a nice enhancement down the road.